awx/ui/src/screens/Job/JobOutput/useJobEvents.js

```diff
@@ -131,11 +137,11 @@ export function jobEventsReducer(callbacks, isFlatMode, enqueueAction) {
     return state;
   }
 
-  function _addRootLevelEvent(state, event) {
+  function _addRootLevelEvent(state, event, isCollapsed = false) {
```

Comment:
Is `isCollapsed` ever used here? I don't see it passed in anywhere in the diff. I would say line 144 should probably just reference `state.isAllCollapsed`, but if it's working without that, maybe neither change is necessary?